Python/courses.md

```diff
@@ -7,4 +7,9 @@
 - [Python with Django](https://goo.gl/DVM0m5) - Django over the python.
 - [Programming fundamental with python](https://goo.gl/JEdLMw) - Intermediate level course on python data structure and basic topics.
 - [Developing Scalable app with Python](https://goo.gl/EwWZDW) - How to use Google app engine along with python to create scalable webApp. 
+- A micro-web-framework of python _flask --- http://flask.pocoo.org/
```

Comment:
This could be included in tools or frameworks, the formal framework guide or documentation should not be included in courses. :smile: